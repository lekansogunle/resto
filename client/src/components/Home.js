import React, { Fragment } from "react";
import HowItWorks from "./HowItWorks";
import Menu from "./Menu";

const Home = () => {
  return (
    <Fragment>
      <HowItWorks />
      <Menu />
    </Fragment>
  )
}

export default Home;