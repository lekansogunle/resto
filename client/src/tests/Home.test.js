import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('simple case', () => {
  expect(2+2).toEqual(4);
})

test('Home, renders home page', () => {

  const component = renderer.create(<Home />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});