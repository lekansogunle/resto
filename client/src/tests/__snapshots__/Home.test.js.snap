// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home, renders home page 1`] = `
Array [
  <div
    className="ui massive steps"
  >
    <div
      className="step"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="add to cart icon"
        onClick={[Function]}
      />
      <div
        className="content"
      >
        <div
          className="title"
        >
          Order
        </div>
      </div>
    </div>
    <div
      className="step"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="payment icon"
        onClick={[Function]}
      />
      <div
        className="content"
      >
        <div
          className="title"
        >
          Pay
        </div>
      </div>
    </div>
    <div
      className="step"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="truck icon"
        onClick={[Function]}
      />
      <div
        className="content"
      >
        <div
          className="title"
        >
          Delivery
        </div>
      </div>
    </div>
  </div>,
  <div />,
]
`;
